Arm/Insts/Common.lean

```diff
@@ -55,6 +55,32 @@ def AddWithCarry (x : BitVec n) (y : BitVec n) (carry_in : BitVec 1) :
   let V := if signExtend (n + 1) result = signed_sum then 0#1 else 1#1
   (result, (make_pstate N Z C V))
 
+/-- When the carry bit is `0`, `AddWithCarry x y 0 = x + y` -/
```

Comment:
The number of lemmas seem to grow in this file. Should we break it into Sections? Better ideas?

Reply:
@pennyannn I think we should add sections, indeed, but I'd prefer to do this in a subsequent PR where I add all the sections in one go! I made an issue to track this: https://github.com/leanprover/LNSym/issues/139